### PR TITLE
Use NGFD for tones.

### PR DESCRIPTION
**Pull request by jusa**

### CODE REVIEW

File: modem/tones.c [hunk 1]
```diff
@@ -52,23 +57,66 @@ struct _ModemTonesPrivate
...
 static void
 modem_tones_init(ModemTones *self)
 {
   self->priv = G_TYPE_INSTANCE_GET_PRIVATE(
     self, MODEM_TYPE_TONES, ModemTonesPrivate);
+  self->priv->connection = dbus_g_bus_get(DBUS_BUS_SYSTEM, NULL);
   self->priv->timer = g_timer_new();
-  self->priv->proxy =
-    dbus_g_proxy_new_for_name(dbus_g_bus_get(DBUS_BUS_SYSTEM, NULL),
-      "com.Nokia.Telephony.Tones",
```
Comment (thp) [inline modem/tones.c]: Are these strings defined in some header files that we can use instead of directly here?

**Comment by thp** [comment]
Thanks for removing the spectacle/yaml cruft :)

File: modem/tones.c (hunk 1 above)
Comment (jusa) [inline modem/tones.c]: What do you mean? Old interface stuff was removed on those lines?

Comment (thp) [inline modem/tones.c]: Ah yeah, not sure why i wrote that in the first place...

**Comment by thp** [comment]
LGTM.

**Pull request merged**